protogen/docs/GRPC.tmpl

```diff
@@ -0,0 +1,97 @@
+---
+{{- $package := (index .Files 0).Package }}
+title: "{{ $package }}"
+url: /grpc_apis/{{ $package }}
```

Comment:
We should rename the section from "Grpc_apis" to "GRPC apis" by adding a _index.md which can be checked in.